hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/webapp/util/WebAppUtils.java

```diff
@@ -428,6 +429,12 @@ public static String getAggregatedLogURL(String serverHttpAddress,
         user == null || user.isEmpty()) {
       return null;
     }
+    String[] ipAdds = serverHttpAddress.split("//");
+    if (ipAdds.length >= 2 && StringUtils.countMatches(ipAdds[1], ":") > 2) {
```

Comment:
I think this is pretty much NetUtils.isIPV6Address

Reply:
fixed